content/initiatives/workplan-sudo-rs.md

```diff
@@ -4,14 +4,21 @@ slug = "workplan-sudo-rs"
 template = "initiatives/workplans/sudo-rs.html"
 +++
 
+### Current work
+
+* Port sudo and su to FreeBSD
+* Improve compatibility with the original sudo utility
+* Support for often-used configuration options in /etc/sudoers
```

Comment:
```suggestion
* Support for often-used configuration options in /etc/sudoers
* Ability to configure /etc/sudoers options on a per-user, per-command, per-host basis.
* Improved environment handling (SETENV)
```
This line item is definitely something we want to do because it's probably the #1 blocker for being a "drop in replacement". SETENV has slightly lesser priority but also something we should table.